src/ducks/filters/SelectDates.styl

```diff
@@ -49,25 +49,42 @@ select-date-height-small = 2.75rem
         right 0
         width auto
         height select-date-height-small
+        padding-left 1rem
+        padding-right 0.875rem
         position fixed
         background white
         z-index 2
         display flex
         justify-content center
         align-content center
         border-bottom: 1px solid silver
+        color slateGrey
+
+    .SelectDates__DateYearSelector
+        flex-grow 1
 
     .SelectDates__chip
         background-color transparent
         margin-bottom 0
         margin-right 0
         padding 0
         height 100%
+        width 100%
+
+        > span:not([class])
```

Comment:
I wanted to target `Select` component's root `<span>`, but there was two issues : 

* The `className` prop in this component is for its `<select>` element
* So I targeted the `<span>`s, but the `Chip.Separator` is also a span. So I only targeted spans that don't have a `class` attribute.

Reply:
I think it would be clearer with a `wrappedClassName` property or something like that. Anyway if you prefer this, I think a comment could help understand :)

Reply:
Thanks, that's better lke this :+1: